src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -54,5 +55,7 @@ protected Result check() throws Exception {
                 return Result.healthy();
             }
         });
+
+        new DbServer();
```

Comment:
It's already in the `run` method, isn't it? Underneath, it's a DataSource` built for `H2`, on which I have no explicit `close` method.